### ISSUE TITLE
Support use of arbitrary colours, other than predefined ones?

### ISSUE THREAD

**Issue by axvr** [zero-one-group/fxl#51]
Hi, I'm loving this approach to building spreadsheets in Clojure.  I'm wondering if support for using arbitrary colours in cell styles is planned as the [current colour selection](https://github.com/zero-one-group/fxl/blob/develop/src/zero_one/fxl/colours.clj) is fairly limited.  Being able to use RGB hex codes (or any other representation) would be great.

**Comment by zackteo** [comment]
Hi Alex! Glad to know that you are enjoying fxl :) I actually did identify colour as an issue here - https://github.com/zero-one-group/fxl/issues/15 but haven't gotten around to implementing it - have been busy since the university term started. 

@anthony-khong might have some thoughts

**Comment by anthony-khong** [comment]
I agree with everyone here! Sounds like something that fxl should really support. I don't have bandwidth this week, but I'll look into it shortly. Thank you @axvr for pointing it out, and thanks @zackteo for the issues 😄
